cartoframes/data/dataset.py

```diff
@@ -227,17 +225,20 @@ def from_geojson(cls, geojson):
 
             Map(Layer(d))
         """
-        return cls(gdf=load_geojson(geojson), state=cls.STATE_LOCAL)
+        return cls(df=load_geojson(geojson), state=cls.STATE_LOCAL)
 
     @property
     def dataframe(self):
         """Dataset DataFrame"""
         return self._df
 
-    @property
     def geodataframe(self):
```

Comment:
Use a @property here

Reply:
:thinking:

Reply:
table_name and dataframe are using properties. We should be consistent with `geodataframe` too

Reply:
> table_name and dataframe are using properties.

They are props and the methods are getters of the props. In this case, I think this one is not a getter

Reply:
IMO this should be a getter, but if this is a method to get or compute the geodataframe, I would call it get_geodataframe or compute_geodataframe

Reply:
In this case, I would move the get_query method to the Dataset class, just to be consistent

Reply:
> just to be consistent

:+1: I suppose you are referring of having:
- get_geodataframe
- get_query
as methods of the "same level".

I am going to change it

Reply:
+1